### ISSUE TITLE
Couldn't implement it, too many bugs

### ISSUE THREAD

**Issue by mdelgadov** [gonultasmf/MyFinance#1]
Hi, I loath to be the bearing of bad news, but couldn't implement it.

I was able to load the devexpress packs, but after that, everything went t_ts up.

I tried to move it to Maui 9 without success...
This is very attractive to me, because I'm doing a dev in Maui.Toolkit.Markup and wanted to look in your Markup library...

Having said that, any pointers will be really appreciated.

After installing, in Android I got this error:
MainActivity.cs

    namespace MyFinance
    {
        [Activity(Theme = "@style/Maui.SplashTheme", MainLauncher = true, ConfigurationChanges = ConfigChanges.ScreenSize | ConfigChanges.Orientation | ConfigChanges.UiMode | ConfigChanges.ScreenLayout | ConfigChanges.SmallestScreenSize | ConfigChanges.Density)]
        public class MainActivity : MauiAppCompatActivity
        {
        }
    }
Line 8: MauiAppCompatActivity
The type or namespace name 'MauiAppCompatActivity' could not be found (are you missing a using directive or an assembly reference?)

MainApplication.cs

namespace MyFinance
{
    [Application]
    public class MainApplication : MauiApplication
    {
        public MainApplication(IntPtr handle, JniHandleOwnership ownership)
            : base(handle, ownership)

The type or namespace name 'MauiApplication' could not be found (are you missing a using directive or an assembly reference?)

and in IOS:
In IOS appdelegate.ce
[Register("AppDelegate")]
    public class AppDelegate : MauiUIApplicationDelegate

The type or namespace name 'MauiUIApplicationDelegate' could not be found (are you missing a using directive or an assembly reference?)

I'm already in SDK9, maybe it's not supported?

Thanks in advance for your help.

**Comment by gonultasmf** [comment]
Hi,
I pulled the project to a computer with Windows operating system that I have never used before. When I started it with .Net 8 version, it was successful. Then, when I updated the versions in the branch called **`_net9-version`** and tried it, I got successful results.

`The type or namespace name 'MauiAppCompatActivity' could not be found (are you missing a using directive or an assembly reference?)` Such errors are caused by Visual Studio not being able to find MAUI references, independent of FmgLib.MauiMarkup.

First of all, let me ask you this: When you create a Maui application from scratch, does the application stand up successfully?

Secondly, are there any reference errors in the views currently under the Views folder in the project?
